examples/asyncio/wsgi-server.py

```diff
@@ -534,23 +533,9 @@ def readline(self, hint=None):
     def readlines(self, hint=None):
         """
         Called by the WSGI application to read several lines of data.
-
-        This method is really pretty stupid. It rigorously observes the
-        ``hint`` parameter, and quite happily returns the input split into
-        lines.
         """
-        # This method is *crazy inefficient*, but it's also a pretty stupid
-        # method to call.
         data = self.read(hint)
-        lines = data.split(b'\n')
-
-        # Split removes the newline character, but we want it, so put it back.
-        lines = [line + b'\n' for line in lines]
-
-        # Except if the last character was a newline character we now have an
-        # extra line that is just a newline: pull that out.
-        if lines[-1] == b'\n':
-            lines = lines[:-1]
+        lines = data.splitlines(True)
```

Comment:
```suggestion
        lines = data.splitlines(keepends=True)
```

Reply:
Not sure what the implications are regarding Unix vs. Windows line ending when using the library splitlines here.

Reply:
Don't think there is any, I've also compared behaviour with Werkzeug's implementation and it does the same for both types on line endings. https://github.com/pallets/werkzeug/blob/main/src/werkzeug/wsgi.py#L425.

Also doesn't violate any requirements from the PEP itself https://peps.python.org/pep-3333/#input-and-error-streams